Projects/UOContent/Items/Containers/Container.cs

```diff
@@ -4,1013 +4,505 @@
 using Server.Multis;
 using Server.Network;
 
-namespace Server.Items
+namespace Server.Items;
+
+[ManualDirtyChecking]
```

Comment:
This is necessary since we can't codegen the class. Eventually MUO will throw an error without this. It doesn't do anything other than to tell the dirty tracker (when in strict mode) to ignore this class.